bot/core/bot.py

```diff
@@ -26,7 +26,7 @@ async def on_ready(self) -> None:
 
             try:
                 self.pool = await asyncpg.create_pool(**config.DATABASE)
-            except:
+            except Exception:
```

Comment:
We probably don't need to except for everything here, there should be a very specific error which occurs here if the database is missing, you should find that error and use that directly instead of excepting for everything which is generally a bad practice until it's really needed

Reply:
Yeah, I was lazy to check. Will do it tomorrow